src/components/scenes/CryptoExchangeQuoteScene.tsx

```diff
@@ -184,31 +184,38 @@ export const CryptoExchangeQuoteScene = (props: Props) => {
   }
   return (
     <SceneWrapper hasTabs hasNotifications padding={theme.rem(0.5)}>
-      {({ insetStyle }) => (
+      {({ insetStyle, undoInsetStyle }) => (
         <>
           <SceneHeader title={lstrings.title_exchange} underline withTopMargin />
-          <ScrollView contentContainerStyle={[insetStyle, styles.container]}>
-            <LineTextDivider title={lstrings.fragment_send_from_label} lowerCased />
-            {showFeeWarning ? <AlertCardUi4 title={lstrings.transaction_details_fee_warning} type="warning" /> : null}
-            <ExchangeQuote quote={selectedQuote} fromTo="from" showFeeWarning={showFeeWarning} />
-            <LineTextDivider title={lstrings.string_to_capitalize} lowerCased />
-            <ExchangeQuote quote={selectedQuote} fromTo="to" />
-            <PoweredByCard iconUri={getSwapPluginIconUri(selectedQuote.pluginId, theme)} poweredByText={exchangeName} onPress={handlePoweredByTap} />
-            {selectedQuote.isEstimate ? (
-              <AlertCardUi4 title={lstrings.estimated_quote} body={lstrings.estimated_exchange_message} type="warning" onPress={handleForEstimateExplanation} />
-            ) : null}
-            {selectedQuote.canBePartial ? (
-              <AlertCardUi4
-                title={lstrings.can_be_partial_quote_title}
-                body={lstrings.can_be_partial_quote_message}
-                type="warning"
-                onPress={handleCanBePartialExplanation}
-              />
-            ) : null}
-
-            <Slider parentStyle={styles.slider} onSlidingComplete={doShift} disabled={pending} showSpinner={pending} />
-            {renderTimer()}
-          </ScrollView>
+          <View style={{ ...undoInsetStyle, marginTop: 0 }}>
+            <ScrollView contentContainerStyle={[{ ...insetStyle, paddingTop: 0 }, styles.container]}>
```

Comment:
You don't need to add a `View` here. The `ScrollView` itself takes a `style` prop.

As a side note, all the builtin react-native lists take a `style` prop, but the `FlashList` does not. This is another reason to move away from `FlashList`